src/components/population-space.tsx

```diff
@@ -0,0 +1,60 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./population-space.sass";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class PopulationSpaceComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    return (
+      <div className="population-space">
+        {this.RenderLeftPanel()}
+        {this.RenderRightPanel()}
+      </div>
+    );
+  }
+
+  private RenderLeftPanel() {
```

Comment:
Should probably be lowercased

Reply:
Yes!  I got too caught up with the uppercase on the exported React component name.  Will change.